Updated Models section (#1025)
ACCESS-Hive/access-hive.github.io: docs/models/access_models/index.md

# ACCESS Models

> ACCESS models are computer programs that represent major components of the Earth system (e.g., atmosphere, land surface, ocean, sea ice, etc.) using complex mathematical equations. These components can be connected through a coupler to create ACCESS models, which are used to simulate past or future conditions as well as idealised experiments.

Reviewer: This is my attempt to make this information a bit more comprehensive for potential new users. I also think we should make it clear here that these are the ACCESS-NRI ACCESS models - there are other ACCESS models that exist that we don't support/develop in any way.

```suggestion
ACCESS models are computer programs that represent key components of the Earth's climate system, including the atmosphere, oceans, land surface, and sea ice. The models use complex mathematical equations to simulate past, present, and future climate conditions, as well as idealised scenarios. Different ACCESS models are configured with varying combinations of Earth system components to address specific research needs, enabling tailored simulations of various aspects of the climate system. These models are developed through collaboration with international climate modeling efforts, but are specifically developed to run on Australia's high-performance computing (HPC) systems, and so are tailored for researchers based in Australia.

The models below are those supported by ACCESS-NRI.
```

Author: I agree we should make it clear that the models are those ACCESS models are those that ACCESS-NRI supports. 
A general considerations on your suggestion is that I would avoid making this introduction too long, as it might deter people from reading it. Below I suggest some modifications to your suggestion to try and improve conciseness and clarity:

> ... to simulate past, present, and future climate conditions, ...

Saying "climate" here is imprecise as Regional models (e.g., ACCESS-rAM3) are often used for weather conditions as well. 
Should we say "weather and climate" instead?

> Different ACCESS models are configured with varying combinations of Earth system components to address specific research needs, enabling tailored simulations of various aspects of the climate system.

I would shorten this bit by removing the last part:
"""
Different ACCESS models are configured with varying combinations of Earth system components to address specific research needs.
"""

> These models are developed through collaboration with international climate modeling efforts, but are specifically developed to run on Australia's high-performance computing (HPC) systems, and so are tailored for researchers based in Australia.
> The models below are those supported by ACCESS-NRI.

I suggest a more concise version:
"""
Developed in collaboration with international climate modeling institutions, these models are optimized for Australia’s high-performance computing (HPC) systems and tailored for Australian researchers. 

Below is a list of ACCESS models supported by ACCESS-NRI.
"""

<div class="card-container">
    <!-- ACCESS-CM -->
    <a href="access-cm/" class="horizontal-card">
        <div class="card-image-container">
            <img src="/assets/model-config-logos/configurations-without-titles/access-cm.png" class="white-background img-contain"></img> 
        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS-CM</span>
            <span>

>                 ACCESS-CM is a global climate model that includes the atmosphere, land, ocean and sea-ice components, and produces physical climate simulations.

Reviewer: ```suggestion
                A global climate model that includes the atmosphere, land, ocean and sea-ice components, and produces physical climate simulations
```

Reviewer: Because by removing the model name, they are no longer full sentences. 😊

            </span>
        </div>
    </a>
    <!-- ACCESS-ESM -->
    <a href="access-esm/" class="horizontal-card">
        <div class="card-image-container">
            <img src="/assets/model-config-logos/configurations-without-titles/access-esm.png" class="white-background img-contain"></img> 
        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS-ESM</span>
            <span>

>                 ACCESS-ESM is a global climate model with additional land and ocean biogeochemical components to simulate both the physical climate and global biogeochemical cycles.

Reviewer: ```suggestion
                A global climate model with additional land and ocean biogeochemical components to simulate both the physical climate and global biogeochemical cycles
```

Author: Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/commit/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852

            </span>
        </div>
    </a>
    <!-- ACCESS-OM -->
    <a href="access-om/" class="horizontal-card">
        <div class="card-image-container">
            <img src="/assets/model-config-logos/configurations-without-titles/access-om.png" class="white-background img-contain"></img> 
        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS-OM</span>
            <span>

>                 ACCESS-OM is a global coupled ocean and sea ice model used to understand both physical and biogeochemical processes in the ocean.

Reviewer: ```suggestion
                A global coupled ocean and sea ice model used to understand both physical and biogeochemical processes in the ocean
```

Author: Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852

            </span>
        </div>
    </a>
    <a href="access-ram/" class="horizontal-card">
        <div class="card-image-container">
            <img src="/assets/model-config-logos/configurations-without-titles/access-ram.png" class="white-background img-contain"></img> 
        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS-rAM</span>
            <span>

>                 ACCESS-rAM is an implementation of the UKMO Regional Nesting Suite that allows high-resolution regional atmosphere modelling.

Author: Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852
